src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php

```diff
@@ -125,14 +137,25 @@ protected function generateDataForRoot(BundleEntryReaderInterface $reader, $temp
     protected function generateDataForMeta(BundleEntryReaderInterface $reader, $tempDir)
     {
         $rootBundle = $reader->read($tempDir, 'root');
+        $metadataBundle = $reader->read($tempDir, 'metadata');
 
         $this->regionCodes = array_unique($this->regionCodes);
 
+        $alpha2ToAlpha3 = $this->generateAlpha3($metadataBundle);
+
         sort($this->regionCodes);
+        $unIndexed = $this->regionCodes;
+
+        $this->regionCodes = [];
+        foreach ($unIndexed as $alpha2Code) {
+            $alpha3code = $alpha2ToAlpha3[$alpha2Code];
+            $this->regionCodes[$alpha3code] = $alpha2Code;
```

Comment:
this alters the keys in `Countries::getCountryCodes()` (im not sure we should do it), whereas for languages this is compiled separate

Reply:
I thought it would be useful to have the 3 letter codes as keys.

Reply:
we ask for a list of alpha2 codes (`getCountryCodes`), i wouldnt mix&match both data sets

Reply:
Well, I was thinking about the symmetry with the `getAlpha3Codes` method where the alpha2 codes are keys.

It also makes it easier to implement `getAplha3Name` and `alpha3CodeExists`.

Reply:
The returnvalue from `getCountryCodes` is now the same as before.